docs/site/Relation-generator.md

```diff
@@ -0,0 +1,106 @@
+---
+lang: en
+title: 'Relation generator'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Relation-generator.html
+summary: summary
+---
+
+# Relation generator
+
+### Prerequisites
+
+Important: Before running this generator, you must create an application using
+the application generator. Then you must run the command from the root directory
+of the application. The models involved in the relation must exist before
+running this generator. The datasource involved in the relation must exist
+before running this generator. The repositories involved in the relation must
+exist before running this generator.
+
+{% include content/generator-create-app.html lang=page.lang %}
+
+### Synopsis
+
+## Creates a new LoopBack4 relation.
```

Comment:
I believe other doc files (e.g. https://github.com/strongloop/loopback-next/blob/51a47332aef88bed9e522485ff2ded42e39c4dd4/docs/site/Application-generator.md)  don't create a new section inside synopsis. Please remove this line entirely, the command is already described in the next paragraph below.

Reply:
Line was removed.